playbooks/bind_config.yaml

```
@@ -0,0 +1,34 @@
+---
+- name: Manage Bind9 configurations
+  hosts: bind_servers
+  become: true
+
+  tasks:
+    - name: Synchronize Bind9 configurations
+      ansible.builtin.copy:
+        src: "{{ item }}"
+        dest: /etc/bind/
+        owner: root
+        group: bind
+        mode: '0644'
+        force: true
+      loop: '{{ lookup("ansible.builtin.fileglob", playbook_dir + "/../../../../../../bind/*", wantlist=True) }}'
```

This path should be configurable in the client config (each repo could have a different directory structure), so it should be a variable. And the default would be better anchored to the inventory instead, as the number of directories to go back could change too.
```suggestion
      loop: '{{ lookup("ansible.builtin.fileglob", bind_config__conf_files, wantlist=True) }}'
```

And then, you could make that variable default to this:
```yaml
bind_config__conf_files: '{{ inventory_dir }}/bind/*.conf'
```

---

Following the comment, we revised this. However, the path had to be updated.

```
bind_config__conf_files: '{{ inventory_dir }}/../../bind/*'
```

- The inventory path is: `sysadmin/ansible/inventory` so 
- Not all files tracked end with `.conf`